src/Models/Services/Utils.ts

```diff
@@ -412,3 +412,8 @@ export const getStoredEditorModelPositionsData = () => {
     const oatData = getStoredEditorData();
     return oatData && oatData.modelPositions ? oatData.modelPositions : [];
 };
+
+// Returns the string value of a key if the object is not a string already
+export const getStringFromObject = (value) => {
+    return typeof value === 'string' ? value : Object.values(value)[0];
+};
```

Comment:
There is a similar function already in the PropertyEditor utils file. Perhaps we can promote that this utils file? And use that instead?